PROJECT

```diff
@@ -1,6 +1,6 @@
 domain: grafana.com
 layout:
-- go.kubebuilder.io/v3
+- go.kubebuilder.io/v4
```

Comment:
Can you point me to the docs for this file?

Reply:
Sure, you can see the detailed steps here:

https://book.kubebuilder.io/migration/manually_migration_guide_gov3_to_gov4

Step: [Migrate the layout version into the PROJECT file](https://book.kubebuilder.io/migration/manually_migration_guide_gov3_to_gov4#migrate-the-layout-version-into-the-project-file)